### DISCUSSION TITLE
[@types/toastify-js] definition missing hideToast method

### DISCUSSION THREAD

**Discussion by pavlexander** [DefinitelyTyped/DefinitelyTyped#60413] [Issues with a @types package] [answered]
in pure js I used toastify as:

```js
      import Toastify from "toastify-js";

      var toast = Toastify({
        text: "Update is pending",
        duration: 3000,
        close: false,
        stopOnFocus: true,
        onClick: function () {
          toast.hideToast(); // see this line
        },
      });
```

After migration to `typescript` I get an error
> S2339: Property 'hideToast' does not exist on type '{ showToast(): void; }'.

If we go to the `toastify-js` source we can find the `hideToast` method there:

https://github.com/apvarun/toastify-js/blob/572517040fae6a7f8be4a99778dacda9c933db45/src/toastify.js#L301

however the definition is missing this method:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/11d8cdcb3dd6408412dc8d74898e82fead85398e/types/toastify-js/index.d.ts#L61

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
Fixing here: #60414
